web/crux-ui/locales/en/registries.json

```diff
@@ -32,7 +32,10 @@
     "gitlab": "Add your GitLab Registry.",
     "github": "Add your GitHub Registry.",
     "google": "Add your Google Registry.",
-    "unchecked": "Add a registry without authentication. This type does not provide image nor tag browsing."
+    "unchecked": "Add a registry without authentication. This type does not provide image nor tag browsing.",
+    "patInfo": "Learn more about personal access token",
```

Comment:
please use descriptive keys like: `learnMorePat` and `learnMoreRegistry`

Reply:
Resolved.